### PR TITLE
Update Bazel invoker to stream output to log.

### PR DESCRIPTION
**Pull request by vadimberezniker**
For the prober we don't get any Bazel output if the probe times out and
gets killed.

Note that this will also affect tests that invoke Bazel.

---

**Version bump**: None

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Comment by vadimberezniker** [comment]
@bduffany I'll wait for your LGTM before merging.

**Review by bduffany — APPROVED** [review]
Nice

**Pull request merged**
